prismic-model/js/parts/heading.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+export default function heading(label: string = 'Title') {
```

Comment:
should this be `heading1`? Or take a number from 1–6 to be passed into the config?

Reply:
It's an interesting piece - we only ever convert headings to text, so it doesn't seem to make a difference - I imagine we might want to add an HTMLHeading at some point, but we don't have a use case for it.